### PR TITLE
Chore: Restore some removed UI components to reduce BC surface area

### PR DESCRIPTION
**Pull request by leeoniya**
this restores the feasible things into the graveyard to minimize breaking changes.

https://github.com/grafana/grafana/pull/84420#issuecomment-1996325919

### CODE REVIEW

File: packages/grafana-ui/src/graveyard/GraphNG/utils.test.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import {
   createTheme,
+  DashboardCursorSync,
   DataFrame,
   DefaultTimeZone,
+  // EventBusSrv,
```
Comment (baldm0mma) [inline packages/grafana-ui/src/graveyard/GraphNG/utils.test.ts:6]: Is this needed?

**Review by baldm0mma — COMMENTED** [review]
The code looks good. Not really sure how to test this manually on my end?

File: packages/grafana-ui/src/graveyard/GraphNG/utils.test.ts (hunk 1 above)
Comment (leeoniya) [reply packages/grafana-ui/src/graveyard/GraphNG/utils.test.ts:6]: nope

**Review by mckn — APPROVED** [review]
LGTM! Thanks for doing this 🙏🏻

**Review by codeincarnate — APPROVED** [review]
Looks good!

**Pull request merged**